plugins/CorePluginsAdmin/angularjs/form-field/form-field.directive.js

```diff
@@ -148,7 +148,7 @@
 
         function getTemplate(field) {
             var control = field.uiControl;
-            if (control === 'password') {
+            if (control === 'password' || control === 'url' || control === 'search' || control === 'email') {
```

Comment:
Yes, it is not shown nicely in the UI. I would presume some materialize issue. I'm supporting it technically but not mentioning it.